frappe/public/js/frappe/desk.js

```diff
@@ -257,6 +257,10 @@ frappe.Application = Class.extend({
 
 	set_rtl: function () {
 		if (["ar", "he"].indexOf(frappe.boot.lang) >= 0) {
+			var ls = document.createElement('link');
```

Comment:
This part of the code was there before I added my part. This changes can be done in an another commit.